### ISSUE TITLE
Gminer containing virus?

### ISSUE THREAD

**Issue by jopaha** [develsoftware/GMinerRelease#510]
Hello everyone,

just curious if anyone here also got the following warning by Windows Defender:

Win32/Contebrew.A!ml

Affected program: gminer/miner.exe

I have not found a single thing about this on google. It this expected?

**Comment by jopaha** [comment]
I really doubt that I´ve downloaded that. Win32/Contebrew.A!ml is the programm that is reporting miner.exe.

Also, if you upload this file to https://www.virustotal.com/gui/home/upload you will see that it gets flagged by Windows Defender there as well.

**Comment by brgabriel09** [comment]
Yeah.. It's really strange behavior. My MSDefender didn't alerted me about the program. Keep in mind that .exe it's mining on your computer ( with your full permission btw ). Maybe you downloaded from a strange/similar source.

Delete the other files and everything that it's similar, get a scan, restart your computer and try download it from here https://github.com/develsoftware/GMinerRelease/releases . 

Or you can accept as it is and use it as it is.

**Comment by lhulet** [comment]
It's alerting you to a "potentially unwanted app." If you don't want the app, you can delete it. If you installed it purposefully, then it is safe.
